tests/MonologCreator/Processor/RequestIdTest.php

```diff
@@ -0,0 +1,237 @@
+<?php
+namespace MonologCreator\Processor;
+
+/**
+ * Class FormatterTest
+ *
+ * @package MonologCreator\Factory
+ */
+class RequestIdTest extends \PHPUnit_Framework_TestCase
+{
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|\MonologCreator\Processor\RequestId
+     */
+    private $subject = null;
+
+    public function setUp()
+    {
+        parent::setUp();
+    }
+
+    public function testConstructor()
+    {
+        $this->subject =
+            $this->getMockBuilder('MonologCreator\Processor\RequestId')
+                ->setMethods(
+                    array(
+                        '_generateUUID',
+                    )
+                )
+                ->getMock();
+        $this->subject->expects($this->once())
+            ->method('_generateUUID');
+        $this->subject->__construct();
+    }
+
+    public function testConstructorWithUUID()
+    {
+        $this->subject =
+            $this->getMockBuilder('MonologCreator\Processor\RequestId')
+                ->setMethods(
+                    array(
+                        '_generateUUID',
+                    )
+                )
+                ->getMock();
+        $this->subject->expects($this->never())
+            ->method('_generateUUID');
+        $this->subject->__construct('mock');
+    }
+
+    public function testInvoke()
```

Comment:
Only test that the Processor is injecting an `request_id` into the `extra` data of the record.